### PR TITLE
Tire irons and tires redux

### PR DESCRIPTION
**Pull request by pepesito1122**
#### Summary

SUMMARY: Content "Introduce tire iron and wheel fastening quality"

#### Purpose of change

Having only a semi-uncommon item such as a wrench to unfast tires is sort lame. So, I decided to introduce a tire iron as more accessible tool that can be found right alongside jacks for far more convenient tire removal/mechanics training. 

#### Describe the solution

Changed it so that 'vehicle_bolt' has been split into 'vehicle_wrench_1', 'vehicle_wrench_2', and 'vehicle_fasten', as well as introduce the "wheel fastening' quality to the game and attaching it to the tire iron and all other tools with the 'wrenching' quality. 

#### Describe alternatives you've considered

I was thinking about adding an OR statement in the requirements for each item that needed 'vehicle_fasten', but I ultimately decided to give all wrenching tools the wheel fastening quality instead.

#### Testing

Testing how wrenches and tire irons function:

1. Spawn tire iron, wrench and a vehicle.
2. Check to see if 'wheel fastening' was needed for tires.
3. Check to see if 'wrenching' was needed for everything that needed that quality, otherwise it would all need 'wheel fastening'.

Testing tire iron spawns:

1. Spawn a car with the proper item group.
2. Check to see if the item spawned commonly enough. 

#### Additional context

I wanted to give this thing another shot because I kind of just fell out of working this little addition. Hopefully this one pulls through proper.

### CODE REVIEW

**Comment by pepesito1122** [comment]
How can I resolve travis and appveyor, again? I'm not sure if I can

**Comment by Night-Pryanik** [comment]
> How can I resolve travis and appveyor, again? I'm not sure if I can

No need to worry about these, they ain't related to your changes.

File: data/json/items/tool/workshop.json [hunk 1]
```diff
@@ -881,6 +883,23 @@
...
+  {
+    "id": "tire_iron",
+    "type": "TOOL",
+    "name": { "str": "tire iron" },
+    "description": "A steel rod bent 90 degrees with a socket on the shorter end, made to hold the lug nuts of a car's wheels to either fasten or loosen for convenient removal and replacement.  Makes for a decent melee weapon in a pinch.",
+    "weight": "454 g",
+    "longest_side": "46 cm",
+    "volume": "1500 ml",
+    "price": 30,
+    "price_postapoc": 50,
```
Comment (Night-Pryanik) [inline data/json/items/tool/workshop.json:895]: Maybe increase price a bit? `50` means it costs mere 50 cents, that's too little IMO for such a handy tool, esp. for post-apocalypse.

**Comment by Night-Pryanik** [comment]
I'm shocked we didn't have this tool all this time.

File: data/json/items/tool/workshop.json (hunk 1 above)
Comment (pepesito1122) [reply data/json/items/tool/workshop.json]: I bumped up the price from 50 cents to around 2 bucks. It sorta shocks me that the wrench is only a single dollar in post-apoc price, though

**Comment by pepesito1122** [comment]
Okay, that's good to know that Travis and App shouldn't matter to me. Thanks

File: data/json/items/tool/workshop.json (hunk 1 above)
Comment (Night-Pryanik) [reply data/json/items/tool/workshop.json]: I think the price should be like ten times higher. Low cost of wrench is an oversight/error, and it should be fixed too.

Comment (pepesito1122) [reply data/json/items/tool/workshop.json]: Right, that's fair

Comment (pepesito1122) [reply data/json/items/tool/workshop.json]: Made it 20 bucks post apoc now

**Comment by pepesito1122** [comment]
So, when can my little pull request can be accepted? I believe ive done everything I needed and wanted for this pull

**Comment by Night-Pryanik** [comment]
Please be patient. There are more than one hundred other PRs waiting for review. Your PR will be reviewed in due time, sooner or later.

**Comment by pepesito1122** [comment]
Of course, thank you for the closure. Please take your time

File: data/json/requirements/vehicle.json [hunk 2]
```diff
@@ -1,8 +1,20 @@
 [
   {
-    "id": "vehicle_bolt",
+    "id": "vehicle_fasten",
```
Comment (Maleclypse) [inline data/json/requirements/vehicle.json:3]: This change is breaking Aftershock which will break tests for everyone.  You need to also make this change in Aftershock items that used `vehicle_bolt` as well as any in Magiclysm if there are any.

**Review by Maleclypse — CHANGES_REQUESTED** [review]
The travis errors at the end need to be fixed since if they aren't they will cause Travis tests to fail on every build. Thanks!

File: data/json/requirements/vehicle.json (hunk 2 above)
Comment (pepesito1122) [reply data/json/requirements/vehicle.json:3]: Right, I'll hop on that

Comment (pepesito1122) [reply data/json/requirements/vehicle.json:3]: Alrighty then, everything seems to work. Thanks for reminding me about aftershock and magi

**Review by Maleclypse — APPROVED** [review]

**Comment by actual-nh** [comment]
Eventually, if different quality levels of WHEEL_FAST could make a difference, a couple of different designs - lower WHEEL_FAST but also some PRY/PRYING quality, vs a 4-way one with higher WHEEL_FAST, would be nice.

**Comment by pepesito1122** [comment]
Yeah, that sounds like a good idea. Like, heavier duty wheels would require higher wheel fastening quality, plus a system in which a higher quality tool would make doing a process less time consuming

**Comment by NoahGoldFox** [comment]
Nice to has a change making stuff abit easier :p also tire-irons are a pretty iconic weapon in post-apocalypse stuff so this ish nice

**Pull request merged**